src/api/ratelimits.mdx

```diff
@@ -5,7 +5,7 @@ sidebar_order: 3
 
 Sentry rate limits every API request made to prevent abuse and resource overuse. The limit is applied to each unique combination of caller and endpoint.
 
-The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected.
+The rate limit follows a fixed window approach. Requests are counted into time buckets, determined by the window size. If the number of requests from a caller exceeds the number of allowed requests within a window, then that request will be rejected. We restrict both how frequently a request is made and how many requests are occurring within same time period.
 
 While there is a default rate limit of **40 requests per second**, each endpoint can have their own maximum number of requests and window size. You can track your rate limit usage by looking at special headers in the response.
```

Comment:
This is going to be confusing now.  We might just need to have separate sections on this page for RPS and CRL.

Reply:
Specifically the `40 requests per second`